### PR TITLE
docs(readme): add zh-tw readme translation

### PR DESCRIPTION
**Pull request by jacklee814**
## Summary

Based on the current README, a Zh-TW translation has been added!

Also updated the count of rules to 270 for other languages for consistency.

## Test Plan

N/A.

### CODE REVIEW

File: packages/@biomejs/biome/README.zh-TW.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+<!-- Insert new entries lexicographically by language code.
+     For example given below is the same order as these files appear on page:
+     https://github.com/biomejs/biome/tree/main/packages/%40biomejs/biome -->
+<div align="center">
+
+[हिन्दी](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.hi.md) | [English](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.md) | 繁體中文 | [简体中文](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.zh-CN.md) | [日本語](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.ja.md) | [Português do Brasil](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.pt-BR.md) | [한글](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.kr.md)
+
+</div>
+
+**Biome** 是一個高效能的 Web 項目工具鏈，旨在提供開發工具以維持這些項目的健康。
```
Comment (lucasweng) [inline packages/@biomejs/biome/README.zh-TW.md:39]: We can translate "projects" as "專案“, as the [Chinese (Traditional) Language Pack for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-zh-hant) also uses "專案" for "projects". (See [source](https://github.com/microsoft/vscode-loc/blob/main/i18n/vscode-language-pack-zh-hant/translations/main.i18n.json#L13339))

```suggestion
**Biome** 是一個高效能的 Web 專案工具鏈，旨在提供開發工具以維持這些專案的健康。
```

File: packages/@biomejs/biome/README.zh-TW.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+**Biome** 旨在支持現代 Web 開發的 [所有主要開發語言][language-support]。
+
+**Biome** [不需要 Node.js](https://biomejs.dev/guides/manual-installation/) 即可運行。
+
+**Biome** 擁有一流的 LSP 支持，配備了能完整保留原文的先進解析器和頂級的錯誤修復能力。
+
+**Biome** 整合了以前分離的工具功能。基於共享基礎構建，讓我們能夠為程式碼處理、錯誤顯示、並行工作、快取記憶體和配置提供一致的體驗。
+
+閱讀更多關於我們的[項目理念][biome-philosophy]。
```
Comment (lucasweng) [inline packages/@biomejs/biome/README.zh-TW.md:90]:
```suggestion
閱讀更多關於我們的[專案理念][biome-philosophy]。
```

File: packages/@biomejs/biome/README.zh-TW.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+**Biome** 採用 [MIT 授權](https://github.com/biomejs/biome/tree/main/LICENSE-MIT) 或 [Apache 2.0 授權](https://github.com/biomejs/biome/tree/main/LICENSE-APACHE)，並根據 [貢獻者公約行為準則](https://github.com/biomejs/biome/tree/main/CODE_OF_CONDUCT.md) 進行管理。
+
+## 資金支持
+
+你可以通過不同的方式支持這個項目
+
+### 項目贊助和資金支持
+
+你可以通過 [Open Collective](https://opencollective.com/biome) 或 [GitHub Sponsors](https://github.com/sponsors/biomejs) 贊助或資助這個項目。
```
Comment (lucasweng) [inline packages/@biomejs/biome/README.zh-TW.md:100]:
```suggestion
你可以通過不同的方式支持這個專案

### 專案贊助和資金支持

你可以通過 [Open Collective](https://opencollective.com/biome) 或 [GitHub Sponsors](https://github.com/sponsors/biomejs) 贊助或資助這個專案。
```

**Review by lucasweng — COMMENTED** [review]
Thanks for the translation work! I've left a few word suggestions.

File: packages/@biomejs/biome/README.zh-TW.md (hunk 1 above)
Comment (jacklee814) [reply packages/@biomejs/biome/README.zh-TW.md]: Thanks for your suggestions and sources!

**Review by lucasweng — APPROVED** [review]

**Review by ematipico — APPROVED** [review]

**Pull request merged**